### ISSUE TITLE
Version 1.3.15.1010: Prompt outputs hostname from envvar in cmd PROMT variable.

### ISSUE THREAD

**Issue by Pathduck** [cmderdev/cmder#2346]
### Purpose of the issue
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question

### Version Information
Version 1.3.15.1010

OS Name Microsoft Windows 10 Pro
Version 10.0.18363 Build 18363
System Type x64-based PC

### Description of the issue
I have a custom prompt in Windows cmd:
`[%computername%] $p$_$+$g$s`

After the 1.3.15 update of Cmder this seems to be picked up by Cmder so its prompt will look like this:
```
d:\bin\Cmder-new

(CRAPSTATION) λ
```

Note the extra line and the hostname before the lambda.
This was not happening in Version 1.3.14.982.

This is a clean install of cmder_mini.zip unpacked in a new directory for testing.

I tried to have a compare of clink.lua but `cmder_prompt` is unchanged.

**Comment by daxgames** [comment]
Just downloaded 1.3.14.  See screenshot.

![image](https://user-images.githubusercontent.com/7318053/87491891-49078b00-c617-11ea-9d56-4bb602fc9a0c.png)

Does your 1.3.14 version have any `*.lua` files in the `%cmder_root%\config` folder?

**Comment by Pathduck** [comment]
@daxgames Thanks for the reply. 

That's the thing - no .lua files in the config folder, as I had no need to customise the prompt earlier. 
I just tested with a clean 1.3.14 and then I do get the Hostname prompt... very strange :/

I have however added a "my_prompt.lua" to the Cmder_new folder to test if custom prompts worked. But I rather prefer to just use the default Cmder prompt.

I did some compares and found the following in my user-ConEmu.xml:
```
			<value name="EnvironmentSet" type="multi">
				<line data="set PATH=%ConEmuBaseDir%\Scripts;%PATH%"/>
				<line data="set PROMPT="/>
			</value>
```

The `set PROMPT` line does not exist in the new clean 1.3.14. 
Apparently this line comes from `vendor\conemu-maximus5\conEmu.xml`. This is set in Startup > Environment in the conEmu settings. Can't remember having set that line myself, maybe it's been inherited over time in the local config.

So that appears to be a working solution at least.
